dts/riscv/telink/telink_b92.dtsi

```diff
@@ -230,9 +230,9 @@
 		};
 
 		usbd: usbd@80100800 {
-			compatible = "telink,b91-usbd";
-			reg = <0x80100800 0x40>;
-			interrupts = <19 1>, <20 1>, <21 1>, <22 1>, <23 1>, <36 1>, <47 1>;
+			compatible = "telink,b9x-usbd";
+			reg = <0x80100800 0x50>;
```

Comment:
@TelinkKrystian
Why did you change from 0x40 to 0x50?
In the B91 datasheet, the offset is 0x00-0x3F